pephub/routers/auth/base.py

```diff
@@ -128,6 +176,10 @@ def callback(
         headers={"Authorization": f"Bearer {x['access_token']}"},
     ).json()
 
+    # append the github access_token to the user data
+    u["gh_access_token"] = x["access_token"]
+    u["gh_refresh_token"] = x["refresh_token"]
```

Comment:
Delete it, right?
it's breaking code

Reply:
Yes, not sure how that ended back there...